# Grid Map Parameters
gridmap_size : 50
resolution : 0.3
# Grid Map Values
UNKNOWN : -1
FREE_SPACE : 0
OBSTACLE : 100
# Ray Casting parameters
distance_margin : 0.5
angular_resolution : 0.25
